alerts/kube_apiserver.libsonnet

```diff
@@ -18,14 +18,16 @@ local utils = import 'utils.libsonnet';
           {
             alert: 'KubeAPIErrorBudgetBurn',
             expr: |||
-              sum(apiserver_request:burnrate%s) > (%.2f * %.5f)
+              sum(apiserver_request:burnrate%s) by (%s) > (%.2f * %.5f)
```

Comment:
Do you think it is worth using named variables here? Takes a moment to make sure the order is correct.

Reply:
Yeah, I actually had that same experience adding in the `by` clause. I tried to keep the disruption minimal that was why I did not change the input for the substitution.